README.md

```diff
@@ -27,14 +27,14 @@ For the gRPC client and server please follow these requirements:
 - support [RFC 3986 URI scheme](https://github.com/grpc/grpc-go/issues/1911); lookout-sdk includes helpers for this:
   - go: using `pb.ToGoGrpcAddress` and `pb.Listen`.
   - python: using `lookout.sdk.grpc.to_grpc_address`.
+- use insecure connection:
+  - currently lookout expects to use insecure gRPC connections
```

Comment:
Let people now that it's provided by `pb.DialContext` helper (as done with gRPC message size)
```suggestion
  - currently lookout expects to use insecure gRPC connections, as provide by `pb.DialContext`
```

Reply:
sorry to nitpick, typo in @dpordomingo's suggestion: as **provided** by

Reply:
:point_up: many thanks @carlosms 
FIXED :dagger:

Reply:
fixed